docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -0,0 +1,77 @@
+
+## Deprecation and removal of the old Coverage engine status checks
+
+Currently, Codacy sends coverage data to your Git provider from both the old and new Coverage engines. We will deprecate and eventually remove the old status checks on the following dates:
+
+<table>
+  <thead>
+    <th>Date</th>
+    <th>Event</th>
+    <th>What to expect</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>February 14th, 2024</td>
+      <td>Deprecation of the old Coverage status check</td>
+      <td>
+        <p>From this day on, the new checks will no longer be marked <strong>beta</strong> and will be the official status checks for Codacy Coverage. The old check will be marked <strong>deprecated</strong> but will keep working.</p>
+        <p>If on this date you are still using the old check to block merging pull requests on GitHub, please <a href="#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github">update your Git provider settings</a>.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>April 18th, 2024</td>
+      <td>Brownout of the old Coverage status check</td>
+      <td>
+        <p>Codacy will not send the old status check on this day.</p>
+        <p>This will help you confirm that your setup has been updated correctly and lower the risk of disruptions when the old status check is removed.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>June 5th, 2024</td>
+      <td>Removal of the old Coverage status check</td>
+      <td>
+        <p>Codacy will stop sending the old status check.</p>
+        <p>If on this date you are still using the old check to block merging pull requests on GitHub, you will no longer be able to merge them. To fix this, <a href="#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github">follow the instructions below</a>.</p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## If you are using the old Coverage status check to block merging pull requests on GitHub
+
+If you are using the old status check to block merging pull requests on GitHub, [edit the branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#editing-a-branch-protection-rule) of your pull requests' target branch under the section **Require status checks to pass before merging**:
+
+![New Coverage status checks GitHub](../images/ala-695-update-status-checks-github.png)
+
+## Differences in coverage metrics between the old and new Coverage engines
+
+You may notice some differences in the coverage metrics reported by the old and new Coverage engines. This may happen because the new Coverage engine calculates coverage metrics considering all the files included in the coverage report, while the old Coverage engine ignores some files:
+
+-   The old Coverage engine ignores any files on a coverage report that aren't also modified by the associated pull request.
```

Comment:
@nicklem it is not about the the modification, is more about 'existing' what do you think about this change?
I would add the change, and then improve this if/when need

```suggestion
-   The old Coverage engine ignores any files on a coverage report that aren't present on the repository on that given commit
```